bin/_mocha

```diff
@@ -191,16 +191,45 @@ Error.stackTraceLimit = Infinity; // TODO: config
 
 var reporterOptions = {};
 if (program.reporterOptions !== undefined) {
-    program.reporterOptions.split(",").forEach(function(opt) {
-        var L = opt.split("=");
-        if (L.length > 2 || L.length === 0) {
-            throw new Error("invalid reporter option '" + opt + "'");
-        } else if (L.length === 2) {
-            reporterOptions[L[0]] = L[1];
-        } else {
-            reporterOptions[L[0]] = true;
-        }
+  var reporterOptionsParser;
+  var multipleReporterFormat = /([^,:]+):{([^}]+)}/g;
+
+  function parseOptions(optionsStr) {
+    var options = {};
+    optionsStr.split(',').forEach(function(opt) {
+      var split = opt.split('=');
+      if (split.length > 2 || split.length === 0) {
+        throw new Error("invalid reporter option '" + opt + "'");
+      }
+
+      if (split.length === 2) {
+        options[split[0]] = split[1];
+      } else {
+        options[split[0]] = true;
```

Comment:
Nice. This wasn't supported in the previous implementation, was it? Or did I miss it?